test/behaviour/connection/database/BUILD

```diff
@@ -51,6 +51,7 @@ java_test(
         "//test/behaviour/config:parameters",
         "//test/behaviour/connection/transaction:steps",
         "//test/behaviour/connection/session:steps",
+        "//test/behaviour/graql:steps",
```

Comment:
is it because one of the scenarios need to do a Graql Define query? https://github.com/graknlabs/behaviour/blob/master/connection/database.feature#L143-L146